### PR TITLE
Update custom "look_at()" method for rigid bodies

### LINKED ISSUE #3835: Custom"look at" method for rigidbody needs updating
**Your Godot version:**
3.2.2

**Issue description:**
As the docs mention a `"custom look_at() method that will work reliably with rigid bodies"` is provided as an example function, however this don't really work as expected and only is able to follow target on a 180 degrees on the x axis.

See: https://stackoverflow.com/questions/59146365/how-to-make-a-rigidbody-look-at-another-spatial

There is no warning or observation about this on the docs. An updated function or explanation on how to make it work as expected ( full rotation from any direction ) will be nice to have. 

**URL to the documentation page (if already existing):**
https://docs.godotengine.org/en/stable/tutorials/physics/rigid_body.html#the-look-at-method

### PR DESCRIPTION
**Pull request by DS-OG**
fixes #3835

### CODE REVIEW

File: tutorials/physics/rigid_body.rst [hunk 1]
```diff
@@ -56,10 +56,10 @@ Here is a custom ``look_at()`` method that will work reliably with rigid bodies:
     {
         private void LookFollow(PhysicsDirectBodyState state, Transform3D currentTransform, Vector3 targetPosition)
         {
-            var upDir = new Vector3(0, 1, 0);
-            var curDir = currentTransform.Basis * new Vector3(0, 0, 1);
+            var upDir = new Vector3.UP;
+            var curDir = currentTransform.Basis.z;
             var targetDir = currentTransform.Origin.DirectionTo(targetPosition);
-            var rotationAngle = Mathf.Acos(curDir.X) - Mathf.Acos(targetDir.X);
+            var rotationAngle = curDir.signed_angle_to(targetDir, up_Dir);
```
Comment (AThousandShips) [inline tutorials/physics/rigid_body.rst:62]:
```suggestion
            var rotationAngle = curDir.SignedAngleTo(targetDir, upDir);
```

File: tutorials/physics/rigid_body.rst [hunk 2]
```diff
@@ -56,10 +56,10 @@ Here is a custom ``look_at()`` method that will work reliably with rigid bodies:
     {
         private void LookFollow(PhysicsDirectBodyState state, Transform3D currentTransform, Vector3 targetPosition)
         {
-            var upDir = new Vector3(0, 1, 0);
-            var curDir = currentTransform.Basis * new Vector3(0, 0, 1);
+            var upDir = new Vector3.UP;
+            var curDir = currentTransform.Basis.z;
```
Comment (AThousandShips) [inline tutorials/physics/rigid_body.rst:60]:
```suggestion
            var curDir = currentTransform.Basis.Z;
```

File: tutorials/physics/rigid_body.rst [hunk 3]
```diff
@@ -56,10 +56,10 @@ Here is a custom ``look_at()`` method that will work reliably with rigid bodies:
     {
         private void LookFollow(PhysicsDirectBodyState state, Transform3D currentTransform, Vector3 targetPosition)
         {
-            var upDir = new Vector3(0, 1, 0);
-            var curDir = currentTransform.Basis * new Vector3(0, 0, 1);
+            var upDir = new Vector3.UP;
```
Comment (AThousandShips) [inline tutorials/physics/rigid_body.rst:59]:
```suggestion
            var upDir = Vector3.Up;
```

**Comment by skyace65** [comment]
This was superseded by #8179. We appreciate the effort though and hope you'll contribute in the future if you find something to fix.

**Pull request closed without merging**